Simplenote/StorageSettings.swift

```diff
@@ -29,10 +29,14 @@ class StorageSettings: NSObject {
         }
         return sharedStorageURL
     }
+
+    let journalModeDisabled = [Constants.journalMode: Constants.journalSetting]
```

Comment:
(Thinking) perhaps this class should only expose the URL(s), and this detail should be private to both SharedStorageMigrator and CoreDataStorage